test/unified_format.py

```diff
@@ -288,11 +297,123 @@ def parse_bulk_write_result(result):
     }
 
 
+def parse_client_bulk_write_models(models):
```

Comment:
We already parse collection.bulk_write arguments into models here:
https://github.com/mongodb/mongo-python-driver/blob/f0e025a127ef5f7bd3f4511eeacb0c8298aed737/test/utils.py#L1264-L1268

Can we reuse that code and remove parse_client_bulk_write_models?